v2/client/src/components/pages/survey/Survey.js

```diff
@@ -5,13 +5,7 @@ import Header from '../../common/Header';
 
 import Questions from './Questions';
 
-import {
-  SurveyQs,
-  SurveyHeader,
-  SessionDetails,
-  Form,
-  Disclaimer,
-} from './Survey.style';
+import { SurveyQs, SurveyHeader, SessionDetails, Form } from './Survey.style';
 
 // formState will be the object where we store survey responses
 // as the participant answers the questions
```

Comment:
better export default the class at the bottom of the file, to be consistent with other files